deployment/articles/03-installing-a-search-engine/02-elasticsearch/01-elasticsearch-intro.markdown

```diff
@@ -75,22 +76,31 @@ Elasticsearch in remote mode, but @product@ can't connect to Elasticsearch, chec
 
 **Cluster name:** 
-: The value of the `cluster.name` property in `elasticsearch.yml` must match the
-`clusterName` property you configured for Liferay's Elasticsearch connector.
+: The value of the `cluster.name` property in `elasticsearch.yml` must match
+the `clusterName` property configured in the @product@ Elasticsearch connector.
 
 **Transport address:** 
-: The value of the `transportAddresses` property in the Elasticsearch connector configuration must
-contain at least one valid host and port where an Elasticsearch node is running. If @product@ is running in
-embedded mode, and you start a standalone Elasticsearch node or cluster, it
-detects that port `9300` is taken and switches to port `9301`. If you then set
-Liferay's Elasticsearch connector to remote mode, it continues to look for
-Elasticsearch at the default port (`9300`).
+: The value of the `transportAddresses` property in the Elasticsearch connector
+configuration must contain at least one valid host and port where an
+Elasticsearch node is running. If @product@ is running in embedded mode, and
+you start a standalone Elasticsearch node or cluster, it detects that port
+`9300` is taken and switches to port `9301`. If you then set Liferay's
+Elasticsearch connector to remote mode, it continues to look for Elasticsearch
+at the default port (`9300`).
 
 The following articles cover the Liferay Connector to Elasticsearch's
 configuration options in more detail.
 
 **Cluster Sniffing (Additional Configurations):**
-: When integrating @product@ with an Elasticsearch cluster containing multiple and different "node" [types](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/modules-node.html#modules-node), Liferay's Elasticsearch connector may throw `NoNodeAvailableException` in the console log if no `data` node is available on the specified transport addresses. This may happen for example when only `master` type node addresses are configured. This is due to the ["cluster sniffing feature"](https://www.elastic.co/guide/en/elasticsearch/client/java-api/6.5/transport-client.html) of the Elasticsearch Transport Client that Liferay's connector is using internally and it is enabled by default:
-   >If, for instance, you initially connect to a master node, after sniffing, no further requests will go to that master node, but rather to any data nodes instead. The reason the transport client excludes non-data nodes is to avoid sending search traffic to master only nodes.
-   
-To disable cluster sniffing, add `client.transport.sniff: false` to the `additionalConfigurations` property in Liferay's Elasticsearch connector configuration.
+: Elasticsearch clusters can have multiple node 
+[types](https://www.elastic.co/guide/en/elasticsearch/reference/6.5/modules-node.html#modules-node).
+[Cluster sniffing](https://www.elastic.co/guide/en/elasticsearch/client/java-api/6.5/transport-client.html), 
+enabled by default enabled by default in the @product@ connector, looks for
```

Comment:
duplicated words

Reply:
No, it's just an echo in my text editor :smile:

Reply:
Fixed - thanks!

Reply:
"of course" it's just your editor :stuck_out_tongue_closed_eyes: